force-app/main/default/flows/map_Two_feild_of_lead_and_opportunity.flow-meta.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why?

Reply:
because standard and custom field are not allowed for map so i use flow